README.md

```diff
@@ -57,8 +57,6 @@ These scripts can all be run from the root level of the repo:
   - Lints Sass using stylelint
 - `yarn test:e2e`
   - Runs end to end tests
-- `yarn test:e2e packages/core/Autocomplete`
-  - Runs a single end to end test, this example runs Autocomplete
```

Comment:
could this be updated to `yarn test:e2e packages/cmsds/Autocomplete` ?

Reply:
i tried it, and i dont think it works currently.